### PR TITLE
Add missing columns to `gwctl get policies`

### LINKED ISSUE #2807: Add missing columns to `gwctl get policies`
Currently some columns(e.g. `AGE`) are missing from the `gwctl get policies` output. Fill the missing gap as per the latest required columns:

screenshot

(Ref: https://deploy-preview-2723--kubernetes-sigs-gateway-api.netlify.app/geps/gep-2722/#output-formats)

### PR DESCRIPTION
**Pull request by jongwooo**
**What type of PR is this?**

/kind feature
/area gwctl
/cc @gauravkghildiyal

**What this PR does / why we need it**:

Add missing columns to `gwctl get policies`.

| Output Columns | Description                                 | Visibility |
|----------------|---------------------------------------------|------------|
| `NAME`           | Name of the Policy                          |            |
| `KIND`           | The kind of policy in the form <kind.group> |            |
| `TARGET NAME`    | Name of the resource the policy applies to  |            |
| `TARGET KIND `   | The kind of target resource in the form     |            |
| `POLICY TYPE`    | Type of policy (Inherited or Direct)        |            |
| `AGE`            | Age of the Policy                           |            |

**Which issue(s) this PR fixes**:

Fixes #2807

**Does this PR introduce a user-facing change?**:

```release-note
Add missing columns to `gwctl get policies`
```

### CODE REVIEW

**Comment by jongwooo** [comment]
I'll work on injecting the fack clock during testing.

File: gwctl/pkg/printer/policies.go [hunk 1]
```diff
@@ -40,20 +43,24 @@ func (pp *PoliciesPrinter) Print(policies []policymanager.Policy) {
 	})
 
 	tw := tabwriter.NewWriter(pp.Out, 0, 0, 2, ' ', 0)
-	row := []string{"POLICY NAME", "POLICY KIND", "TARGET NAME", "TARGET KIND", "POLICY TYPE"}
+	row := []string{"NAME", "KIND", "TARGET NAME", "TARGET KIND", "POLICY TYPE", "AGE"}
```
Comment (gauravkghildiyal) [inline gwctl/pkg/printer/policies.go:46]: Lets also configure the `KIND` column to output the resource type in the format `<KIND>.<GROUP>`. Example: `timeoutpolicy.foo.com`

**Review by gauravkghildiyal — COMMENTED** [review]
Thanks @jongwooo! Just one small request and rest all good :) Thank you for the patience.

**Pull request merged**